src/services/iiif-parser.js

```diff
@@ -122,3 +121,41 @@ export function getMediaFragment(uri) {
     return undefined;
   }
 }
+
+export function getCanvas(uri) {
```

Comment:
Maybe this should be renamed `getCanvasId` to distinguish it from getting the whole canvas object?